deploy/crs/v2/kieapp_rhdm_production-immutable.yaml

```diff
@@ -16,4 +16,9 @@ spec:
             uri: https://github.com/jboss-container-images/rhdm-7-openshift-image.git
             reference: master
             contextDir: quickstarts/hello-rules/hellorules
+          # uncomment the block below to set custom environment variable for BuildConfig, useful to configure JAVA_OPTS_APPEND or other
+          # properties such as MAVEN_OPTS_APPEND
+          # env:
+          #  - name: "JAVA_OPTS_APPEND"
+          #    value: "-Dmy.custom.property=prop-test"
```

Comment:
this seems like a good candidate for a snippet instead

Reply:
alright, added to snippet.